### PR TITLE
Fix Travis CI

### PR DESCRIPTION
**Pull request by Juanlu001**
Supersedes #182 by @tomkralidis.

### CODE REVIEW

**Comment by Juanlu001** [comment]
```
src/pydap/tests/test_server_devel.py ./home/travis/.travis/functions: line 104:  5904 Segmentation fault      (core dumped) pytest --cov=src/pydap/
```

:angry:

**Comment by Juanlu001** [comment]
Not willing to spend time fixing Python 3.4, which is EOL already, so picking up on #176 as well.

**Comment by Juanlu001** [comment]
This might be the culprit: https://stackoverflow.com/a/53764657/554319

**Comment by Juanlu001** [comment]
Victory :heart_eyes: This is ready to merge!

**Comment by tomkralidis** [comment]
Thanks much!

**Comment by Juanlu001** [comment]
My pleasure! Thanks for the quick response :grin:

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 language: python
 
+sudo: required
```
Comment (hugovk) [inline .travis.yml]: `sudo` is no longer needed: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Comment (Juanlu001) [reply .travis.yml]: Thanks @hugovk! Do you want to make a PR for it?

Comment (hugovk) [reply .travis.yml]: Sure, please see PR https://github.com/pydap/pydap/pull/193!

**Pull request merged**